src/sequenceParser/Sequence.hpp

```diff
@@ -206,28 +239,12 @@ class Sequence
 	std::string string() const;
 
 public:
-	std::string _prefix; ///< filename prefix
-	std::string _suffix; ///< filename suffix
-	/**
-	 * @brief Number max of common padding used to enumerate the sequence
-	 * @note For fixed sequences, it is equal to the padding
-	 * @note Useful for sequence with a variable or an unknown padding
-	 * unknown padding = when no frame begins with a '0' padding character
-	 * seq.101.jpg
-	 * seq.102.jpg
-	 * seq.103.jpg
-	 * variable padding = when not all frames have the same padding
-	 * seq.0101.jpg
-	 * seq.0100.jpg
-	 * seq.099.jpg
-	 */
+	std::string _prefix;
+	std::string _suffix;
+
```

Comment:
I moved it to the getMaxPadding method. See commit c1560e42004d8fb46685239ff4676d6614b5881b